pkg/cvo/cvo.go

```diff
@@ -62,30 +71,38 @@ type Operator struct {
 	apiExtClient  apiextclientset.Interface
 	eventRecorder record.EventRecorder
 
-	syncHandler func(key string) error
+	syncHandler                 func(key string) error
+	availableUpdatesSyncHandler func(key string) error
 
-	clusterOperatorLister oslistersv1.ClusterOperatorLister
+	// payloadDir is intended for testing. If unset it will default to '/'
+	payloadDir string
+	// defaultUpstreamServer is intended for testing.
+	defaultUpstreamServer string
 
-	crdLister             apiextlistersv1beta1.CustomResourceDefinitionLister
-	deployLister          appslisterv1.DeploymentLister
+	clusterOperatorLister oslistersv1.ClusterOperatorLister
```

Comment:
nit: can we group corresponding lister and listersynced

Reply:
Yes